core/types/transaction_test.go

```diff
@@ -666,23 +667,25 @@ func TestEffectiveGasTipInto(t *testing.T) {
 		tx, _ := SignNewTx(key, signer, txdata)
 
 		var baseFee *big.Int
+		var baseFee2 *uint256.Int
 		if tc.baseFee != nil {
 			baseFee = big.NewInt(*tc.baseFee)
+			baseFee2 = uint256.NewInt(uint64(*tc.baseFee))
 		}
 
 		// Get result from original method
```

Comment:
I know you just modified this test, but it is not clear to me what are we trying to test here.
The "orig" method is calling the "new" method, so not clear to me what is this test for. Shouldn't we compare to expected results instead?